TCMSApp/src/client/app/runs/runs-exec/runs-edit.controller.js

```diff
@@ -0,0 +1,49 @@
+( function() {
+    'use strict';
+
+    angular
+        .module('app.runsExec')
+        .controller('RunsEditController', RunsEditController);
+
+    RunsEditController.$inject = ['$stateParams','$state', 'logger', 'moment'];
+
+    function RunsEditController($stateParams, $state) {
+        var vm = this;
+        vm.run = JSON.parse(JSON.stringify($stateParams.run));//creating of the deep copy
```

Comment:
you shall write above this line:

`//TODO: change this temporary solution of exchanging run data between edit and run tabs`